datastore/elastic_search/query.py

```diff
@@ -1,10 +1,20 @@
+# std imports
+import copy
 from six import string_types
 import re
 import collections
-from lib.nlp.const import TOKENIZER
-from ..constants import ELASTICSEARCH_SEARCH_SIZE, ELASTICSEARCH_VERSION_MAJOR, ELASTICSEARCH_VERSION_MINOR
+
+# 3rd Party Imports
+from elasticsearch import helpers
+
+# Local imports
+from ..constants import ELASTICSEARCH_SEARCH_SIZE, ELASTICSEARCH_VERSION_MAJOR, ELASTICSEARCH_VERSION_MINOR, \
+    ELASTICSEARCH_BULK_HELPER_MESSAGE_SIZE
```

Comment:
change this to `from datastore import constants`
and use `constants.ELASTICSEARCH_SEARCH_SIZE`  for `ELASTICSEARCH_SEARCH_SIZE`